### DISCUSSION TITLE
Wasm Backends support in Wasmer - cast your vote!

### DISCUSSION THREAD

**Discussion by syrusakbary** [wasmerio/wasmer#3839] [Polls]
We recently improved our codebase to add support for more Wasm backends/runtimes (starting with [JavascriptCore](https://wasmer.io/posts/wasmer-3.3-and-javascriptcore))

This opens new exciting possibilities as users can now easily switch and compare the best backend suited for their environment.

**Comment by timscodebase** [comment]
My Vote

**Comment by syrusakbary** [comment]
Hey @everyone, we just launched Wasmer 5.0-rc1 with support for V8, WASMI and WAMR via the `wasm_c_api`.

Feel free to try it out with:

```bash
curl https://get.wasmer.io -sSfL | sh -s "v5.0.0-rc.1"
```

Also, specific `v8`, `wasmi` and `wamr` releases can be found on the release page:
https://github.com/wasmerio/wasmer/releases/tag/v5.0.0-rc.1

Stay tuned for the upcoming 5.0 announcement

**Reply by bashor** [reply]
🤩 Wow! Congrats!

**Reply by bashor** [reply]
@syrusakbary, could you please share an example of how to use it with v8? Does this mean it will support GC and EH when run with v8?

**Reply by syrusakbary** [reply]
Hey @bashor, yes GC and EH will be supported in Wasmer thanks to V8 integration. You just need to use the `v8` feature when using the `wasmer` crate

**Reply by bashor** [reply]
@syrusakbary is it possible to use v8 from CLI?

**Comment by syrusakbary** [comment]
Wasmer v5.0 is now officially released with V8 and Wasmi support. Closing this discussion now

Here's the blogpost for the announcement: https://wasmer.io/posts/introducing-wasmer-v5
